src/demos/graphlib.test.js

```diff
@@ -1,10 +1,13 @@
 const graphlib = require("graphlib");
-const { intersection, isEmpty } = require("lodash");
+const { isEmpty } = require("lodash");
 
-const findOrphanedQuestion = (nodesWithoutOutEdges, nodesWithoutInEdges) =>
-  !isEmpty(intersection(nodesWithoutInEdges, nodesWithoutOutEdges));
+const isOrphaned = (id, firstQuestion, graph) => {
+  if (isEmpty(graph.inEdges(id)) && id !== firstQuestion) {
+    return id;
+  }
```

Comment:
These tests are more testing the functionality of the library with context to project, than testing the project itself; there's a separate test for that. 

The old test to find an orphaned child was incorrect, so I updated it. 

It's not using the full function also because the real function works slightly differently, and requires a bit more logic, but for the purposes of this test I figured I could simplify the function and mock out some of it.

Yeah, I can have it return true or false instead. Either or works; I'll modify it to return that instead.